packages/fluentui/docs/src/examples/components/List/Types/ListExampleSelectable.shorthand.tsx

```diff
@@ -25,6 +25,14 @@ const items = [
   },
 ];
 
-const ListExampleSelectable = () => <List selectable defaultSelectedIndex={0} items={items} />;
+const ListExampleSelectable = () => (
+  <Box
+    styles={({ theme: { siteVariables } }) => ({
+      backgroundColor: siteVariables.colorScheme.default.background4,
```

Comment:
having seen this fix, I no longer understand the original issue. Is the List intended to be used with a white background ? if it is then I don't see how this is a fix

Reply:
the list color is correct. It is not intended to be used in white background in teams-v2, so I added a background color around the example

Reply:
> It is not intended to be used in white background in teams-v2

Can you add that to the PR description ? that's what was confusing (to me anyway) since the original issue is written like this is a bug with fluent, not the docsite.

Also this example seems similarly 'broken': 
![image](https://user-images.githubusercontent.com/20744592/123764611-3a1bf600-d8c5-11eb-94ba-64cd44e8ad18.png)

Reply:
Good catch! I'll update this example as well. Also PR description is updated to be clear about this is only a docsite issue :)